src/include/connection.h

```diff
@@ -302,6 +302,8 @@ struct __wt_connection_impl {
 	WT_CONDVAR	*stat_cond;	/* Statistics log wait mutex */
 	const char	*stat_format;	/* Statistics log timestamp format */
 	WT_FSTREAM	*stat_fs;	/* Statistics log stream */
+	/* Statistics log print json formatted tables */
```

Comment:
Can you reword this comment - it makes it sound like somewhere I'd check to see if we are printing json formatted tables, wheras it's recording state during printing of json formatted tables.